src/style.css

```diff
@@ -0,0 +1,135 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.card {
+  background-color: var(--white-color);
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  cursor: pointer;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  /* text-align: center; */
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+  /* text-align: left; */
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+  /* background-color: gray; */
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) center no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+/* .stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+} */
```

Comment:
it's should work for all modifiers
and check the checklist 
<img width="581" alt="image" src="https://user-images.githubusercontent.com/99331793/201314145-9b96b941-6035-4fe6-a29d-5c4acae703eb.png">